devices/ICub/src/ICub.cpp

```diff
@@ -0,0 +1,1067 @@
+/*
+ * Copyright (C) 2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include "ICub.h"
+
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Network.h>
+#include <yarp/os/Time.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/os/Property.h>
+#include <yarp/dev/IEncoders.h>
+#include <yarp/dev/IAxisInfo.h>
+#include <yarp/os/ResourceFinder.h>
+
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/ModelIO/ModelLoader.h>
+#include <iDynTree/KinDynComputations.h>
+
+#include <assert.h>
```

Comment:
cassert

Reply:
don in https://github.com/robotology/wearables/pull/25/commits/42eba1b291ebfda8a27629c4a999cf1225bdca30